MonoGame.Framework.Content.Pipeline/TargetPlatform.cs

```diff
@@ -102,7 +102,12 @@ public enum TargetPlatform
         /// <summary>
         /// WebAssembly and Bridge.NET
         /// </summary>
-        Web
+        Web,
+
+        /// <summary>
+        /// All desktop versions using Vulkan.
+        /// </summary>
+        DesktopVK,
```

Comment:
@mrhelmut @harry-cpp this is the plan correct?

Reply:
Yeah